### DISCUSSION TITLE
Not able to find peers [not enough connections]

### DISCUSSION THREAD

**Discussion by rohanjacin** [libp2p/js-libp2p#2569] [Q&A]
Hi,

I've been trying to start a Light node, but am unable to find any peers/connections. The following is my setup 
Node: v18.0.0
@waku/sdk : 0.0.25

The following is my code 
`import { createLightNode,
		 waitForRemotePeer,
		 Protocols } from "@waku/sdk";

class Waku {
	constructor () {
		this.node = null; 
	}

	async start () {
		this.node = await createLightNode({ deafultBootstrap: true});

		await this.node.start();
		console.log("Started");
		await waitForRemotePeer(this.node);
		console.log("Waited");
	} 
}

const waku = new Waku();
waku.start();
console.log("Done");`

![Screenshot from 2024-05-29 17-05-37](https://github.com/libp2p/js-libp2p/assets/166788978/5d7b769f-5125-4b09-8da4-ccbe7ed44025)

**Comment by rohanjacin** [comment]
I also tried starting the node in the browser with help from the @waku/react guide, it gives me the following error.

The code is 
`import logo from './logo.svg';
import './App.css';
import { useWaku } from "@waku/react";

function App() {
  const { node, error, isLoading } = useWaku();
  console.log("node:",  node);
  console.log("error:" +  error);
  console.log("isLoading:" +  isLoading);

  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
}

export default App;
`

![Screenshot from 2024-05-29 17-43-48](https://github.com/libp2p/js-libp2p/assets/166788978/dc12b3a9-37d1-450e-8b49-ea8826661569)

**Comment by achingbrain** [comment]
Sorry this went so long without a reply - I hope you managed to find a solution to your problem.

Waku's light client has [it's own peer discovery mechanisms](https://github.com/waku-org/js-waku/blob/master/packages/sdk/src/create/discovery.ts#L15-L19) and doesn't use the standard js-libp2p ones - if they aren't working as you expect them to I would suggest asking on the Waku repo.
